tsconfig.json

```diff
@@ -10,10 +10,11 @@
     "noImplicitThis": true,
     "noUnusedLocals": true,
     "preserveConstEnums": true,
-    "sourceMap": true
+    "sourceMap": true,
+    "stripInternal": true
   },
   "include": [
-    "lib/**/*.ts",
+    "src/**/*.ts",
```

Comment:
I assume this will output JS files to src as well?

Might be good to also immediately specify the files to be packaged in package.json.

Reply:
Currently yes, we can always change that in the future if there are problems with that.

Good idea, I'll add it.